app/views/shared/_not_current_provider_modal.html.erb

```diff
@@ -22,8 +22,13 @@
     <% end %>
 
     <% if options[:draft] %>
-      <%= link_to 'Yes', collection_draft_path(options[:draft_id]), class: 'eui-btn--blue spinner is-invisible', id: 'not-current-provider-view-draft-link' %>
-      <%= link_to 'Yes', edit_collection_draft_path(options[:draft_id], form: options[:draft_form]), class: 'eui-btn--blue spinner is-invisible', id: 'not-current-provider-edit-draft-link' %>
+      <% if options[:variable] %>
+        <%= link_to 'Yes', variable_draft_path(options[:draft_id]), class: 'eui-btn--blue spinner is-invisible', id: 'not-current-provider-view-draft-link' %>
```

Comment:
The ids for the links to `view` and `edit` the variable drafts are identical to the ids for links for the collection drafts here. They should be unique, right?

Reply:
Only one set of IDs is rendered onto the page, so they are unique in the browser

Reply:
ah, ok.